### PR TITLE
correct typo in contributing guide step 5

### PR DESCRIPTION
**Pull request by kafui4k**
## Issue Link 🔗: https://github.com/BlackPythonDevs/blackpythondevs.github.io/issues/521

**Issue**: #521 

## Type of Change

- [ ] Bug fix 🐞
- [ ] New feature/page
- [ x] Documentation update
- [ ] Other

## Description 📋

- **What:** Typo in fifth point under "creating an issue", which should read: And that's it! If you "choose" to... and not if you "cose".

- **Why:** This is a fix in the contributing guide to correct a typo.

- **How:** For clarity when reading.

## Checklist ✅

- [x ] Followed the [Code of Conduct](https://github.com/BlackPythonDevs/blackpythondevs.github.io?tab=coc-ov-file) and [Contribution Guide](https://github.com/BlackPythonDevs/blackpythondevs.github.io/blob/469afd71943738b7209bfe56d5a4e35394fbaafc/CONTRIBUTING.md)
- [ x] Ran `pre-commit run --all`
- [ x] All tests pass locally
- [ ] Added tests (if applicable)
- [x ] Documentation updated (if applicable)

## Additional Notes & Screenshots

Add any additional notes or comments that might be helpful for the reviewers.

### CODE REVIEW

File: .gitignore
Comment (kjaymiller) [inline .gitignore]:
```suggestion
```

I get this is your local environment but we shouldn't start a precedent of adding custom envfiles

**Review by kjaymiller — CHANGES_REQUESTED** [review]
Able to approve once you approve the suggested change

File: .gitignore
Comment (kafui4k) [reply .gitignore]: Yes, thats right

**Review by kjaymiller — APPROVED** [review]

**Pull request merged**
